packages/stacked_generator/test/form_generator/form_generator_test.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:stacked_generator/src/generators/forms/field_config.dart';
+import 'package:stacked_generator/src/generators/forms/form_util_service.dart';
+import 'package:stacked_generator/src/generators/forms/form_view_config.dart';
+import 'package:test/test.dart';
+
+import 'constant_test_helper.dart';
+
+void main() {
+  group('FormGeneratorTest -', () {
+    group('generateImport -', () {
+      test('when called should generate imports', () {
+        FormUtilService utilService = FormUtilService(
+          formViewConfig: FormViewConfig(
+            viewName: 'Test',
+            fields: [],
+          ),
+        );
+        utilService.generateImports();
```

Comment:
the class name is FormGeneratorTest but you're acctually testing FormUtilService 
the fix I talked about in the previous comments will fix your tests too